bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/InboxResource.java

```diff
@@ -108,6 +108,13 @@ public Response ignore(@PathParam("thingUID") String thingUID) {
         inbox.setFlag(new ThingUID(thingUID), DiscoveryResultFlag.IGNORED);
         return Response.ok().build();
     }
+    
+    @POST
+    @Path("/unignore/{thingUID}")
```

Comment:
Yes - HABmin is using the /ignore endpoint. To be honest - I don’t mind changing it if it’s deemed best…